qiita_pet/handlers/websocket_handlers.py

```diff
@@ -70,3 +72,26 @@ def on_close(self):
         yield Task(self.toredis.unsubscribe, self.channel)
         self.r_client.delete('%s:messages' % self.channel)
         self.redis.disconnect()
+
+
+class SelectSamplesHandler(WebSocketHandler, BaseHandler):
```

Comment:
Error catching, 404 handling, and default_user handling. All the stuff that is built in BaseHandler that I didn't want to re-implement for a websocket handler.